pkg/dispatchcli/cmd/create.go

```diff
@@ -105,17 +109,28 @@ func importFile(out io.Writer, errOut io.Writer, cmd *cobra.Command, args []stri
 				return err
 			}
 			o.Images = append(o.Images, m)
+			fmt.Fprintf(out, "Created %s: %s\n", docKind, *m.Name)
 		case utils.FunctionKind:
 			m := &functionModels.Function{}
 			err = yaml.Unmarshal(doc, &m)
 			if err != nil {
 				return errors.Wrapf(err, "Error decoding function document %s", string(doc))
 			}
+			if strings.HasPrefix(*m.Code, "@") {
+				functionPath := (*m.Code)[1:]
```

Comment:
Shouldn't this be modified to work with the work-dir flag?

Reply:
Could you create a new issue?  Also, I'd like path supported for dependency files (e.g. requirements.txt/package.json)